include/tvm/runtime/data_type.h

```diff
@@ -90,7 +91,10 @@ class DataType {
   /*! \return number of bytes to store each scalar. */
   int bytes() const { return (bits() + 7) / 8; }
   /*! \return number of lanes in the data. */
-  int lanes() const { return static_cast<int>(data_.lanes); }
+  int lanes() const {
```

Comment:
it might be useful to distinguish between lanes and vscale_factor, by having two functions, only allow lanes to corresponds to static vector lanes

Reply:
We can use a different function for scalable vector vscale multiplier

Reply:
Done